README.md

```diff
@@ -34,17 +34,29 @@ mamba install -c conda-forge molfeat
 
 _**Tips:** You can replace `mamba` by `conda`._
 
-_**Note:** We highly recommend using a [Conda Python distribution](https://github.com/conda-forge/miniforge) to install Molfeat. The package is also pip installable if you need it: `pip install molfeat`._
+_**Note:** We highly recommend using a [Conda Python distribution](https://github.com/conda-forge/miniforge) to install Molfeat. The package is also pip installable if you need it: `pip install molfeat`._ 
+
+### Optional dependencies
+Not all featurizers of the Molfeat core package are supported by default. Some featurizers require additional dependencies. If you try to use a featurizer that requires additional dependencies, Molfeat will raise an error and will tell you which dependencies are missing and how to install these. 
+
+- To install `dgl`: run `mamba install -c dglteam dgl`
+- To install `dgllife`:  run `mamba install -c dglteam dgllife`
```

Comment:
`dgllife` is available on conda-forge.

Reply:
The version on `dglteam` is 3 years old and should not be used.

Reply:
fixed for `dgllife`